scripts/eurostat/health_determinants/physical_activity/process.py

```diff
@@ -11,912 +11,180 @@
-            sv_list += df["SV"].to_list()
-
-        final_df = final_df.sort_values(by=['time', 'geo', 'SV', 'observation'])
-        final_df = final_df.drop_duplicates(subset=['time','geo','SV'],\
-            keep='first')
-        final_df['observation'] = final_df['observation'].astype(str)\
-            .str.strip()
-        # derived_df generated to get the year/SV/location sets
-        # where 'u' exist
-        derived_df = final_df[final_df['observation'].astype(str).str.contains(
-            'u')]
-        u_rows = list(derived_df['SV'] + derived_df['geo'])
-        final_df['info'] = final_df['SV'] + final_df['geo']
-        # Adding Measurement Method based on a condition, wherever u is found
-        # in an observation. The corresponding measurement method for all the
-        # years of that perticular SV/Country is made as Low Reliability.
-        # Eg: 2014
-        #   country/AUT
-        #   Percent_AerobicSports_NonWorkRelatedPhysicalActivity_In_Count_Person
-        #   77.3 u,
-        # so measurement method for both 2014 and 2019 years shall be made
-        # low reliability.
-        final_df['Measurement_Method'] = np.where(
-            final_df['info'].isin(u_rows),
-            'EurostatRegionalStatistics_LowReliability',
-            'EurostatRegionalStatistics')
-        final_df.drop(columns=['info'], inplace=True)
-        final_df['observation'] = (
-            final_df['observation'].astype(str).str.replace(
-                ':', '').str.replace(' ', '').str.replace('u', ''))
-        final_df['observation'] = pd.to_numeric(final_df['observation'],
-                                                errors='coerce')
-        final_df = final_df.replace({'geo': COUNTRY_MAP})
-        final_df = final_df.sort_values(by=['geo', 'SV'])
-        final_df['observation'].replace('', np.nan, inplace=True)
-        final_df.dropna(subset=['observation'], inplace=True)
-        final_df.to_csv(self._cleaned_csv_file_path, index=False)
-        sv_list = list(set(sv_list))
-        sv_list.sort()
-        self._generate_mcf(sv_list)
-        self._generate_tmcf()
-
-
-def main(_):
-    input_path = FLAGS.input_path
-    if not os.path.exists(input_path):
-        os.mkdir(input_path)
+    _import_name = "physical_activity"
+
+    _mcf_template = ("Node: dcid:{sv}\n"
+                     "{sv_name}\n"
+                     "typeOf: dcs:StatisticalVariable\n"
+                     "populationType: dcs:Person"
+                     "{denominator}"
+                     "{incomequin}"
+                     "{education}"
+                     "{healthbehavior}"
+                     "{exercise}"
+                     "{residence}"
+                     "{activity}"
+                     "{duration}"
+                     "{gender}"
+                     "{countryofbirth}"
+                     "{citizenship}"
+                     "{lev_limit}"
+                     "{frequency}"
```

Comment:
Can these be derived from keys of `_sv_properties_template` ignoring ones with `_temp` suffix?